docs/source/index.rst

```diff
@@ -51,12 +51,15 @@ The following related subprojects target end users and have separate documentati
   queries.
 - `DataFusion Comet <https://datafusion.apache.org/comet/>`_ is an accelerator for Apache Spark based on
   DataFusion.
+- `DataFusion Ballista <https://datafusion.apache.org/ballista/>`_ is a distributed compute platform built on
```

Comment:
DataFusion Ballista is distributed processing extension for DataFusion 

Would that make more sense?